addons/website/static/tests/tours/link_tools.js

```diff
@@ -176,6 +191,16 @@ wTourUtils.registerWebsitePreviewTour('link_tools', {
         isCheck: true,
     },
     // 7. Create new a link from a URL-like text.
+    {
+        content: "click on Mega menu to hidden Mega menu",
```

Comment:
```suggestion
        content: "click on Mega menu item to hide Mega menu content",
```
I guess that's what you meant?